sormas-backend/pom.xml

```diff
@@ -121,20 +121,28 @@
 		</dependency>
 
 		<dependency>
-			<groupId>info.novatec</groupId>
-			<artifactId>bean-test</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.jboss.weld.se</groupId>
-			<artifactId>weld-se</artifactId>
-		</dependency>
-		<dependency>
-			<groupId>org.apache.deltaspike.core</groupId>
-			<artifactId>deltaspike-core-impl</artifactId>
+			<groupId>de.hilling.junit.cdi</groupId>
+			<artifactId>cdi-test-core</artifactId>
+			<version>3.4.0</version>
+			<scope>test</scope>
+			<exclusions>
+				<exclusion>
+					<artifactId>junit-jupiter-api</artifactId>
```

Comment:
Do not exclude `<artifactId>junit-jupiter-api</artifactId>` here, but add it in the dependency management in the sormas-base/pom.xml to override the version,